src/navbar/index.tsx

```diff
@@ -236,6 +157,123 @@ export function NavBar() {
   );
 }
 
+export function NavBarWebStudio() {
```

Comment:
Would appreciate a more descriptive name. How about `NavBarBroadcast` or `NavBarConnection` or `NavBarStreamer`? Something in that vein - "WebStudio" is the name of the project